doc/en/migration/upgradetoCentreon19.10.rst

```diff
@@ -151,17 +151,17 @@ installation. The main directories to synchronize are:
         lrwxrwxrwx   1 root root      24  1 nov.  17:59 plugins -> /usr/lib/nagios/plugins/
         -rwxr-xr-x   1 root root 1711288  6 avril  2018 cbmod.so
     
-    You can now push poller configuration from Centreon 18.10 whether the distant poller is centreon-engine 18.10 or 1.8.1
+    You can now push poller configuration from Centreon 19.10 whether the distant poller is centreon-engine 19.10 or 1.8.1
```

Comment:
"centreon-engine 19.10 or 1.8.1" -> are you sure

Reply:
Honestly I do not know if Centreon Engine configuration files are and/or should be compatible with earlier versions. I guess that some clarification is needed but I guess that only @lpinsivy can clarify this.

Reply:
According to @Moujimouja and @quanghungb this should be left as it is until further notice.

Reply:
I don't think that you'll be able to push a 19.10 configuration on a 1.8.1 poller.
Until further more information, from LPI, could you remove the " or 1.8.1" part ?

Reply:
Doc update is part of MON-4071, so let it with 1.8.1 until this story modifies this part of the doc